cmd/operator-ui/actions/blackducks.go

```diff
@@ -231,25 +225,20 @@ func (v BlackducksResource) Create(c buffalo.Context) error {
 	log.Infof("create blackduck: %+v", blackduck)
 
 	_, err = util.GetHub(v.blackduckClient, blackduck.Spec.Namespace, blackduck.Spec.Namespace)
-
 	if err == nil {
 		return v.redirect(c, blackduck, fmt.Errorf("blackduck %s already exist", blackduck.Spec.Namespace))
 	}
 
 	_, err = util.GetNamespace(v.kubeClient, blackduck.Spec.Namespace)
-
-	if err == nil {
-		return v.redirect(c, blackduck, fmt.Errorf("namespace %s already exist", blackduck.Spec.Namespace))
-	}
-
-	ns, err := util.CreateNamespace(v.kubeClient, blackduck.Spec.Namespace)
 	if err != nil {
-		return v.redirect(c, blackduck, err)
+		err = util.DeployCRDNamespace(v.kubeConfig, v.kubeClient, util.BlackDuckName, blackduck.Spec.Namespace, blackduck.Name, blackduck.Spec.Version)
+		if err != nil {
+			return v.redirect(c, blackduck, err)
+		}
+		log.Infof("created %s Black Duck instance in %s namespace", blackduck.Name, blackduck.Spec.Namespace)
```

Comment:
Should we slightly change the log message? Only the namespace has been created at this stage so "created namespace %s for Black Duck instance %s" would be more appropriate here.

Reply:
Sorry didn't see your comment. Will fix it in next PR